tests/driver/rust/build.rs

```diff
@@ -175,7 +175,8 @@ fn generate_source(
             Some(testcase.absolute_path.file_stem().unwrap().to_str().unwrap().to_string());
     }
     if source.contains("//no-default-translation-context") {
-        compiler_config.no_default_translation_context = true;
+        compiler_config.default_translation_context =
+            i_slint_compiler::DefaultTranslationContext::None;
```

Comment:
I realized the implementation in the slint-compiler was missing, so this fixes it.